tests/instrumentation-framework/src/main/java/org/hawkular/apm/tests/ProjectVersion.java

```diff
@@ -0,0 +1,24 @@
+package org.hawkular.apm.tests;
```

Comment:
Yes I will move it to the sub package. License is missing, PR was just created to quick review if you agree with my approach.